src/features/utils/aggregate-base.js

```diff
@@ -136,14 +136,14 @@ export class AggregateBase extends FeatureBase {
       } catch (err) {
         // do nothing
       }
-      configure({ agentIdentifier: this.agentIdentifier }, {
+      configure(existingAgent, {
```

Comment:
if `configure(...` has a reference to `existingAgent` already, do we need the third argument?

Reply:
Yes, since agent.js and micro-agent.js passes in a 3rd argument with different defaulted values. I left that unchanged.